src/AutoRest.CSharp/Common/AutoRest/Communication/StandaloneGeneratorRunner.cs

```diff
@@ -70,6 +70,11 @@ public static async Task RunAsync(CommandLineOptions options)
                 throw new InvalidOperationException($"Neither CodeModel.yaml nor cadl.json exist in {outputPath} folder.");
             }
 
+            if (options.ClearOutputFolder)
+            {
+                var keepFiles = new string[] { "CodeModel.yaml", "Configuration.json", "cadl.json" };
```

Comment:
We should keep this files in SDK library. Only for debug. 

Since we clear output folder after generate, we should not need them

Reply:
I believe this is OK because we only write them when the `save-inputs` option is on.
We should not remove them if the option is on, and if the option is not on, we will not have these files.
Therefore adding these files as exceptions makes sense

Reply:
These files are controlled by "save-inputs", not "clear-output-folder".